src/main/java/seedu/address/logic/commands/AddModCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_NAME;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.Module;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddModCommand extends Command {
+
+    public static final String COMMAND_WORD = "addmod";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a module to the address book. "
```

Comment:
Should we use FaculType instead of address book or just keep it as it is?

Reply:
I'll change it to FaculType instead.